### DISCUSSION TITLE
Helper function that returns the payload instead of ray instance

### DISCUSSION THREAD

**Discussion by dbpolito** [spatie/ray#646] [Ideas]
Sometimes in code you just want to wrap way in a value you still want to be untouched, like

```php
    return ray_returning_payload($something);
```

I would like it go to ray but still return `$something`, i thought about doing a PR but couldn't think a good name for this function 😅 

The same behavior laravel `dump` has, it dumps but still return the payload so behavior in code doesn't change.

**Comment by freekmurze** [comment]
You're looking for `ray()->pass($variable)`. It'll show `$variable` in Ray and also return `$variable` 👍  Happy debugging!
